setup.py

```diff
@@ -38,7 +38,7 @@
     python_requires=">=3.6",
     install_requires=[
         "pulp-glue==0.21.0.dev",
-        "click>=8.0.0,<9.0.0",
+        "click>=8.1.0,<8.1.3",
```

Comment:
```suggestion
        "click>=8.1.0,<8.1.4",
```

Reply:
also, is there a reason to up the LB from 8.0.0? If not, let's make smallest-required-change here

Reply:
Ok, lowering LB to 8.0.0